project/Build.scala

```diff
@@ -63,9 +63,9 @@ object DottyJSPlugin extends AutoPlugin {
 }
 
 object Build {
-  val referenceVersion = "3.0.0"
+  val referenceVersion = "3.0.1-RC1"
```

Comment:
For stability purposes, shouldn't we wait until we have `3.0.1`?

Reply:
It might be better, yes.

Reply:
I disagree, the point of doing an RC is to test that it works well, and since not many people are going to test it, it's a good idea that we use it as our own reference compiler to caught issues that would slip through otherwise (e.g., incremental compilation issues).